docs/using-the-jdbc-driver/using-plugins/UsingTheHostMonitoringPlugin.md

```diff
@@ -72,3 +72,17 @@ properties.setProperty("monitoring-socketTimeout", "10");
 > We recommend you either disable the Host Monitoring Connection Plugin or avoid using RDS Proxy endpoints when the Host Monitoring Connection Plugin is active.
 >
 > Although using RDS Proxy endpoints with the AWS Advanced JDBC Driver with Enhanced Failure Monitoring doesn't cause any critical issues, we don't recommend this approach. The main reason is that RDS Proxy transparently re-routes requests to a single database instance. RDS Proxy decides which database instance is used based on many criteria (on a per-request basis). Switching between different instances makes the Host Monitoring Connection Plugin useless in terms of instance health monitoring because the plugin will be unable to identify which instance it's connected to, and which one it's monitoring. This could result in false positive failure detections. At the same time, the plugin will still proactively monitor network connectivity to RDS Proxy endpoints and report outages back to a user application if they occur.
+
+# Host Monitoring Plugin v2
+
+>:warning: This plugin is experimental and should be verified before using it in production environment.
```

Comment:
```suggestion
> [!WARNING] This plugin is experimental and should be verified before using it in production environment.
```

Reply:
I am not sure "should be verified" makes much sense. Let's talk about "tested/

- This plugin is still in experimental stage. That means that users should test the plugin first for their use case scenarios before using it in their production environment.